clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -69,6 +69,7 @@ sealed abstract class DeliverCommand[+CI <: DeliverableIndex](override val index
 
 sealed abstract class PatchCommand[CI <: ClioIndex](val index: CI) extends ClioCommand {
   def metadataLocation: URI
+  def parallelism: Int
```

Comment:
To me this name is unclear. The API called below that accepts this argument calls this attribute `breadth` and describes it as "at most `breadth` substreams are being consumed at any given time".

Maybe `maxParallelStreams`?